src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -98,15 +98,15 @@ public function require(array $packages): array
      */
     public function remove(array $packages): void
     {
-        $this->updateImportMapConfig(false, [], $packages);
+        $this->updateImportMapConfig(false, [], $packages, []);
     }
 
     /**
-     * Updates all existing packages to the latest version.
+     * Updates either all existing packages or the specified ones to the latest version.
      */
-    public function update(): array
+    public function update(array $packages): array
```

Comment:
As this class is not internal, maybe the argument should be optional to avoid a BC break.

Btw, it would even make sense to be able to call `$importManager->update()` to update everything.

Reply:
Adding a required argument is a BC break for callers, not for child classes (for child classes, adding _any_ argument is a BC break)